src/Tizen.System.Session/Session/Session.cs

```diff
@@ -0,0 +1,509 @@
+        {
+            _switchUserWaitHandler?.Invoke(this, new SwitchUserWaitEventArgs(infoNative));
+        }
+
+        private Interop.Session.SubsessionEventCallback _switchUserWaitCB = null;
+
+        private event EventHandler<SwitchUserWaitEventArgs> _switchUserWaitHandler = null;
+
+        private readonly object _switchUserWaitLock = new object();
+
+        /// <summary>
+        /// Event to be invoked when an existing subession user has begun switching to an active state.
+        /// </summary>
+        /// <exception cref="ArgumentException">Session UID of this object is invalid</exception>
+        /// <exception cref="OutOfMemoryException">Out of memory</exception>
+        /// <exception cref="IOException">Internal error</exception>
+        /// <exception cref="UnauthorizedAccessException">Not permitted</exception>
+        /// <exception cref="NotSupportedException">Not supported</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<SwitchUserWaitEventArgs> SwitchUserWait
+        {
+            add
+            {
+                lock (_switchUserWaitLock)
+                {
+                    if (_switchUserWaitHandler == null)
+                        RegisterCallbackForEvent(SessionEventType.SwitchUserWait, _switchUserWaitCB, OnSwitchUserWait);
+                    _switchUserWaitHandler += value;
+                }
+            }
+            remove
+            {
+                lock (_switchUserWaitLock)
+                {
+                    _switchUserWaitHandler -= value;
+                    if (_switchUserWaitHandler == null)
+                        UnregisterCallbackForEvent(SessionEventType.SwitchUserWait, _switchUserWaitCB);
+                }
+            }
+        }
+
+        private void OnSwitchUserCompletion(SubsessionEventInfoNative infoNative, IntPtr data)
+        {
+            _switchUserCompletionHandler?.Invoke(this, new SwitchUserCompletionEventArgs(infoNative));
+        }
+
+        private Interop.Session.SubsessionEventCallback _switchUserCompletionCB = null;
+
+        private event EventHandler<SwitchUserCompletionEventArgs> _switchUserCompletionHandler = null;
+
+        private readonly object _switchUserCompletionLock = new object();
+
+        /// <summary>
+        /// Event to be invoked when an existing subession user is successfully switched to an acvite state.
+        /// </summary>
+        /// <exception cref="ArgumentException">Session UID of this object is invalid</exception>
+        /// <exception cref="OutOfMemoryException">Out of memory</exception>
+        /// <exception cref="IOException">Internal error</exception>
+        /// <exception cref="UnauthorizedAccessException">Not permitted</exception>
+        /// <exception cref="NotSupportedException">Not supported</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<SwitchUserCompletionEventArgs> SwitchUserCompletion
```

Comment:
What about `SwitchUserCompleted` ? 

Please check it
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-events